torch/utils/_benchmark/utils/common.py

```diff
@@ -21,6 +21,8 @@
 _IQR_WARN_THRESHOLD = 0.1
 _IQR_GROSS_WARN_THRESHOLD = 0.25
 
+_CACHE_SPEEDUP_THRESHOLD = 1.5
+_CACHE_SPEEDUP_MIN_SECONDS = 0.000005
```

Comment:
Can you either change this to `5e-6` or add a comment `# 5 us`?

Reply:
"5e-6" sounds kind of scientific... How about 5 / (1000.0 * 1000)?